gcp-jobs/involuntary-dissolutions/src/involuntary_dissolutions/involuntary_dissolutions.py

```diff
@@ -0,0 +1,444 @@
+    db,
+)
+from business_model.models.db import init_db
+from croniter import croniter
+from dissolution_service import InvoluntaryDissolutionService
+from flask import Flask
+from simple_cloudevent import SimpleCloudEvent, to_queue_message
+from sqlalchemy import Date, cast, func
+from sqlalchemy.orm import aliased
+
+from gcp_queue import GcpQueue
+from structured_logging import StructuredLogging
+
+from .config import get_named_config  # pylint: disable=import-error
+from .involuntary_dissolution_types import DissolutionTypes
+from .utils.flags import Flags
+
+gcp_queue = GcpQueue()
+
+flags = Flags()
+
+env = os.getenv("FLASK_ENV", "production")
+def create_app(run_mode=env):
+    """Return a configured Flask App using the Factory method."""
+    app = Flask(__name__)
+    app.config.from_object(get_named_config(run_mode))
+    app.env = run_mode
+    init_db(app)
+
+    if app.config.get("LD_SDK_KEY", None):
+        flags.init_app(app)
+
+    register_shellcontext(app)
+    gcp_queue.init_app(app)
+    app.logger = StructuredLogging(app).get_logger()
+
+    return app
+
+
+def register_shellcontext(app):
+    """Register shell context objects."""
+    def shell_context():
+        """Shell context objects."""
+        return {"app": app}
+
+    app.shell_context_processor(shell_context)
+
+
+def create_invountary_dissolution_filing(business_id: int):
+    """Create a filing entry to represent an involuntary dissolution filing."""
+    business = Business.find_by_internal_id(business_id)
+
+    filing = Filing()
+    filing.business_id = business.id
+    filing._filing_type = CoreFiling.FilingTypes.DISSOLUTION  # pylint: disable=protected-access
+    filing._filing_sub_type = DissolutionTypes.INVOLUNTARY  # pylint: disable=protected-access
+    filing.filing_json = {
+        "filing": {
+            "header": {
+                "date": datetime.now(UTC).date().isoformat(),
+                "name": "dissolution",
+                "certifiedBy": ""
+            },
+            "business": {
+                "legalName": business.legal_name,
+                "legalType": business.legal_type,
+                "identifier": business.identifier,
+                "foundingDate": business.founding_date.isoformat()
+            },
+            "dissolution": {
+                "dissolutionDate": datetime.now(UTC).date().isoformat(),
+                "dissolutionType": "involuntary"
+            }
+        }
+    }
+
+    filing.hide_in_ledger = True
+    filing.save()
+
+    return filing
+
+
+def put_filing_on_queue(filing_id: int, app: Flask):
+    """Send queue message to filer to dissolve business."""
+    try:
+        subject = app.config["FILER_SUBJECT"]
+        msg = {"filing": {"id": filing_id}}
+        app.logger.debug(f"Attempting to place filing on Filer Queue with id {filing_id}")
+        ce = SimpleCloudEvent(
+            id=str(uuid.uuid4()),
+            source=app.config.get("CLIENT_NAME"),
+            subject=subject,
```

Comment:
This shouldn't be set to the GCP topic. It's meant for extra context beyond the 'type' (ie. involuntary-dissolution-job-filing)

Reply:
done